Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -2332,14 +2334,15 @@ public void applyReferralCode(final String code, final BranchReferralInitListene
      * user selected clients.</p>
      *
      * @param builder A {@link io.branch.referral.Branch.ShareLinkBuilder} instance to build share link.
+     * @return The instance of the {@link Dialog} holding the share view.
      */
-    private void shareLink(ShareLinkBuilder builder) {
+    private Dialog shareLink(ShareLinkBuilder builder) {
```

Comment:
The way I see it, there are two ways this will go:
  1. We'll get support requests from new users who duplicate calls to `show` if we leave it.
  1. We'll get support requests from old users who don't know that the need to call `show` if we don't.